cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -805,6 +850,21 @@ func (a *StaticAutoscaler) deleteCreatedNodesWithErrors() (bool, error) {
 		if nodeGroup == nil {
 			err = fmt.Errorf("node group %s not found", nodeGroupId)
 		} else {
+			opts, err := nodeGroup.GetOptions(a.NodeGroupDefaults)
```

Comment:
This `err` changes the following logic `err = nodeGroup.DeleteNodes(nodesToBeDeleted)`.